### PR TITLE
remove a repeat for CVE-2013-0209

### PR DESCRIPTION
**Pull request by GeorgeAnAltx**
(first from start of document, second from 483 line)
image

### CODE REVIEW

**Comment by briandfoy** [comment]
In the description of the report, this says that the vulnerability is "through" 4.38, which I interpret to mean that it includes that version. Do you have any references to indicate that is not the case?

The other changes seems cosmetic. Was there some problem you encountered?

Note that you can send pull requests, and that will automatically have a diff so you don't have to post an image.

File: cpansa/CPANSA-File-Path.yml [hunk 1]
```diff
@@ -149,7 +149,7 @@
     - https://usn.ubuntu.com/94-1/
   reported: 2005-05-02
   severity: ~
-- affected_versions: ">=1.404,<=1.06"
+- affected_versions: "<=1.404,>=1.06"
```
Comment (briandfoy) [inline cpansa/CPANSA-File-Path.yml]: This is a change for a different file?

Comment (briandfoy) [reply cpansa/CPANSA-File-Path.yml]: I've fixed this up in a separate commit. For some reason I wasn't able to cherry pick this.

Comment (GeorgeAnAltx) [reply cpansa/CPANSA-File-Path.yml]: sorry for duplicate commit

> I've fixed this up in a separate commit. For some reason I wasn't able to cherry pick this.

Comment (briandfoy) [reply cpansa/CPANSA-File-Path.yml]: Hey, no problem. I'd rather have a slightly annoying commit than no commit so the problem goes unnoticed. Thanks again.

**Pull request closed without merging**